src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/androids/MinerAndroid.java

```diff
@@ -45,15 +60,16 @@ protected void dig(Block b, BlockMenu menu, Block block) {
 
             if (SlimefunPlugin.getProtectionManager().hasPermission(owner, block.getLocation(), ProtectableAction.BREAK_BLOCK)) {
                 AndroidMineEvent event = new AndroidMineEvent(block, new AndroidInstance(this, b));
-                Bukkit.getPluginManager().callEvent(event);
 
-                if (event.isCancelled()) {
-                    return;
+                // We only want to break non-Slimefun blocks by default, although an addon may want it differently.
+                String blockId = BlockStorage.checkID(block);
+                if (blockId != null) {
+                    event.setCancelled(true);
```

Comment:
This can be simplified

```suggestion
                if (BlockStorage.hasBlockInfo(block)) {
                    event.setCancelled(true);
```